test/lib/headers_test.rb

```diff
@@ -139,6 +139,31 @@ def test_pathname_with_trailing_slash_if_present_with_query_lang_when_trailing_s
       assert_equal('/test/', headers.pathname_with_trailing_slash_if_present)
     end
 
+    # Issue for: https://github.com/WOVNio/wovnrb/issues/156
```

Comment:
I would remove the comment since people can find it from git history, but OK if you want to keep it.